stdlib/public/runtime/ImageInspectionELF.cpp

```diff
@@ -68,8 +68,12 @@ static SectionInfo getSectionInfo(const char *imageName,
   SectionInfo sectionInfo = { 0, nullptr };
   void *handle = dlopen(imageName, RTLD_LAZY | RTLD_NOLOAD);
   if (!handle) {
+#ifdef __ANDROID__
+    return sectionInfo;
+#else
```

Comment:
Why is this relevant to the fatal error patch?  I would prefer this get reviewed separately if it could.